tcp.c

```diff
@@ -423,6 +423,7 @@ tcp_connect(char *server)
 	socklen_t option_len;
 	uint32 option_value;
 	int i;
+	char buf[INET6_ADDRSTRLEN];
```

Comment:
Is this constant available on non-IPv6 systems? And do we care? :)

Reply:
I don't actually know. It's in POSIX, so I figured it was OK.